### PR TITLE
Number fallback on pinned message view

### PR DESCRIPTION
**Pull request by g-harel**
#21 

The `captures/` changes are unrelated to user story. The erased files were originally added by Dan somehow.

### CODE REVIEW

File: src/org/thoughtcrime/securesms/PinnedMessageAdapter.java [hunk 1]
```diff
@@ -120,9 +120,20 @@ public void onClick(View v) {
...
+
+        if (record.isOutgoing()) {
+            viewHolder.recipient.setText(ownMessageName);
+            return;
         }
+
+        String messageSenderName = record.getRecipient().getName();
+
+        if (messageSenderName == null) {
+            messageSenderName = record.getRecipient().getAddress().toString();
```
Comment (veken1199) [inline src/org/thoughtcrime/securesms/PinnedMessageAdapter.java:131]: That's the simplest and most efficient way to complete this story #21

**Review by veken1199 — DISMISSED** [review]

File: src/org/thoughtcrime/securesms/PinnedMessageAdapter.java (hunk 1 above)
Comment (Dan-Yoo) [reply src/org/thoughtcrime/securesms/PinnedMessageAdapter.java:131]: looks good!

**Review by Dan-Yoo — DISMISSED** [review]

File: src/org/thoughtcrime/securesms/PinnedMessageAdapter.java [hunk 2]
```diff
@@ -120,9 +120,20 @@ public void onClick(View v) {
     }
 
     private void setMessageView(MessageRecord record, ViewHolder viewHolder) {
-        if(!record.isOutgoing()) {
-            viewHolder.recipient.setText(record.getRecipient().getName());
+        String ownName = "Me";
```
Comment (erinb90) [inline src/org/thoughtcrime/securesms/PinnedMessageAdapter.java]: This should be defined in strings.xml and referenced here. See https://developer.android.com/guide/topics/resources/string-resource.html

File: src/org/thoughtcrime/securesms/PinnedMessageAdapter.java (hunk 1 above)
Comment (erinb90) [reply src/org/thoughtcrime/securesms/PinnedMessageAdapter.java:131]: 👍

**Comment by g-harel** [comment]
@Radu-Raicea

**Review by Radu-Raicea — APPROVED** [review]

**Comment by Radu-Raicea** [comment]
Everything looks good to me! Ready to merge?

**Comment by g-harel** [comment]
yes

**Pull request merged**